scripts/data_prep/convert_finetuning_dataset.py

```diff
@@ -38,6 +44,10 @@ def parse_args() -> Namespace:
                         default=None,
                         help='Name or import path of function used to preprocess (reformat) the dataset. ' +\
                              'See README for additional details.')
+    parser.add_argument('--data_files',
```

Comment:
clarify docs that this is a 1:1 mapping between splits and file. Also could you test if there are any restrictions on file naming? I know sometimes HF requires that the split name match the filename and stuff like that.

Reply:
> Also could you test if there are any restrictions on file naming?

For data_files it's fine, since you input the file name, not the directory name.